playground/backend/internal/code_processing/code_processing_test.go

```diff
@@ -532,64 +526,7 @@ func TestGetLastIndex(t *testing.T) {
 	}
 }
 
-func Test_setJavaExecutableFile(t *testing.T) {
```

Comment:
I believe, we also need to test all introduced functions like `validateStep`, `compileStep`, etc.

Reply:
Done